bats/tests/helpers/commands.bash

```diff
@@ -80,3 +80,8 @@ rdshell() {
 rdsudo() {
     rdshell sudo "$@"
 }
+WSL_DISTRO=rancher-desktop
```

Comment:
So far we have put variable defaults at the top, before all the function definitions (see e.g. `CONTAINERD_NAMESPACE`, which is conceptually similar.

Reply:
Yeah, to keep things consistent let me move it.